modules/apps/transfer/keeper/forwarding.go

```diff
@@ -48,13 +48,13 @@ func (k Keeper) forwardPacket(ctx sdk.Context, data types.FungibleTokenPacketDat
 }
 
 // ackForwardPacketSuccess writes a successful async acknowledgement for the prevPacket
-func (k Keeper) ackForwardPacketSuccess(ctx sdk.Context, prevPacket channeltypes.Packet) error {
+func (k Keeper) ackForwardPacketSuccess(ctx sdk.Context, prevPacket channeltypes.Packet, forwardedPacket channeltypes.Packet) error {
```

Comment:
```suggestion
func (k Keeper) ackForwardPacketSuccess(ctx sdk.Context, prevPacket, forwardedPacket channeltypes.Packet) error {
```
signature could be updated to drop the double type declaration, but fine as is

Reply:
Forgot about this possibility, done!